### PR TITLE
re-add community marketplace and handle missing dependency

### PR DESCRIPTION
**Pull request by MattDHill**
~~Waiting on Registry changes~~ completed by @elvece

### CODE REVIEW

**Review by elvece — APPROVED** [review]

File: frontend/projects/marketplace/src/types.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export interface MarketplacePkg {
     [id: string]: {
       title: string
       icon: Url
+      'is-local': boolean
```
Comment (Blu-J) [inline frontend/projects/marketplace/src/types.ts:33]: :question: Is this guaranteed to exist?

Comment (Blu-J) [reply frontend/projects/marketplace/src/types.ts:33]: :pleading_face: Could I get any link that any other service is changing to use this.

The only thing that I have found to have this field is the [api fixures ](https://github.com/Start9Labs/embassy-os/pull/2110/files#diff-c96a082192e9e6068d0e16576e235a91fd429993f8d3f203fc64c6c04374c1d8)
Guessing that the service changes are needed, would like the link after changes.

Comment (MattDHill) [reply frontend/projects/marketplace/src/types.ts:33]: It will come from the registry

**Review by elvece — DISMISSED** [review]

**Review by elvece — APPROVED** [review]

File: backend/src/version/v0_3_3_1.rs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                .installed()
+                .expect(db)
+                .await?
+                .marketplace_url()
+                .put(db, &parsed_url)
+                .await?;
+        }
+        Ok(())
+    }
+    async fn down<Db: DbHandle>(&self, db: &mut Db) -> Result<(), Error> {
```
Comment (MattDHill) [inline backend/src/version/v0_3_3_1.rs:65]: @Blu-J are you purposfully not reverting the marketplace_url for community services?

**Review by elvece — APPROVED** [review]

**Pull request merged**